docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+.. image:: images/installintro.png
+   :width: 75%
+   :alt: Diagram illustrating the Flocker CLI and control service and agents.
 
-.. note:: The ``clusterhq-flocker-node`` package is pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
+The CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
+This package also includes the ``flocker-ca`` tool, which you must use to generate certificates for all the Flocker components.
+
+The Flocker agents are installed on any number of hosts (or nodes) in the cluster where your containers will run.
```

Comment:
The image refers to hosts and the rest of the doc refers to nodes, hence this imperfect association of the two. When the image is redone perhaps it could refer to nodes and then this sentence replaced by "nodes" only @davidoliverSP2.